source/geometries/Next1EL.cc

```diff
@@ -1371,7 +1373,7 @@ G4ThreeVector Next1EL::GenerateVertex(const G4String& region) const
       }
   } else if (region == "MUONS") {
     //generate muons sampling the plane
-    vertex = muons_sampling_->GenerateVertex();
+    vertex = muons_sampling_->GenerateVertex("INSIDE");
```

Comment:
Why doesn't this use an enum?

Reply:
Nevermind, I see that this is calling the legacy sampler where it is indeed a string.